docs/data/joy/components/select/select.md

```diff
@@ -52,6 +52,20 @@ Use the `startDecorator` and/or `endDecorator` props to add supporting icons or
 
 {{"demo": "SelectDecorators.js"}}
 
+If you have interactive elements as the select's decorators, call `stopPropagation()` from the mouse down event to prevent the popup from being opened.
```

Comment:
Is it possible to have an interactive element as a selects decorator without breaking its a11y?